### DISCUSSION TITLE
SQL INSERT erro

### DISCUSSION THREAD

**Discussion by etaptecnologia** [balta-io/forum#108] [Tira Dúvidas] [answered]
Msg 2628, Level 16, State 1, Line 1
String or binary data would be truncated in table 'Cursos.dbo.Curso', column 'Nome'. Truncated value: 't'.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @etaptecnologia, como vai?

Este problema ocorre quando você tenta enviar uma string maior que o campo no banco suporta.

Por exemplo, se você definiu `VARCHAR(30)` no banco e mandar uma string com mais de 30 caracteres, vai ter este erro.
